src/requirements/Payment/PaymentRequirement.tsx

```diff
@@ -79,8 +73,7 @@ const PaymentRequirement = (props: RequirementProps): JSX.Element => {
       rightElement={
         props?.rightElement ? (
           <GuildCheckoutProvider>
-            {(satisfiesRequirement && !multiplePayments) ||
-            (poapLinks && poapLinks?.claimed === poapLinks?.total) ? (
+            {satisfiesRequirement && !multiplePayments ? (
```

Comment:
Deleted the mint link availability check here, I'm not sure if we need that or not? What do you think?
In my opinion, if we decide to include a check, we should check if the role has any text-based rewards (NFT, Secret, Unique secret, POAP) & if it has, we should disable the pay button, but still not sure if we need that extra logic here or not. Lmk what do you think!